projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1872,6 +1888,20 @@ protected static function build_sprintf( $args ) {
 	 * @return string
 	 */
 	public function publicize_connections_url( $source = 'calypso-marketing-connections' ) {
+
+		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+		$is_atomic_site = ( new Status\Host() )->is_woa_site();
+
+		$is_wpcom = $is_simple_site || $is_atomic_site;
+
+		if ( ! $is_wpcom && $this->use_admin_ui_v1() && current_user_can( 'manage_options' ) ) {
+			$is_social_active = defined( 'JETPACK_SOCIAL_PLUGIN_DIR' );
+
+			$page = $is_social_active ? 'jetpack-social' : 'jetpack#/sharing';
```

Comment:
This means we'll default to the Social admin page when both plugins are enabled. I'm not sure if that's a problem.

Reply:
I think preferring the standalone plugin is better, isn't it?